ddsc/core/fileuploader.py

```diff
@@ -234,7 +254,11 @@ def upload_async(data_service_auth_data, config, upload_id,
     data_service = DataServiceApi(auth, config.url)
     sender = ChunkSender(data_service, upload_id, filename, config.upload_bytes_per_chunk, index, num_chunks_to_send,
                          progress_queue)
-    return sender.send()
+    try:
+        sender.send()
```

Comment:
This runs in a background process and upload_async now traps exceptions and sends them back to the main process. We no longer return a value from this method.